README.md

```diff
@@ -66,8 +66,19 @@ samples = RunACSampler(num_of_samples, params_AC);
 The sampling function starts by generating the number of required samples and then runs OPF for each of the samples and filter those with feasible OPF solutions. Importing power grid data, grid modifications and solving OPF are all done within [PowerModels.jl](https://github.com/lanl-ansi/PowerModels.jl) framework. Since some of the generated samples might not be feasible, the sample generation continues in an iterative manner until the required number of samples with feasible solution is met. Currently, if more than 60\% of the samples lead to infeasible OPF in the first iteration, the algorithm returns an error to indicate the fact that the choice of parameters might not be suitable for the used grid.    
 
 #### Output Structure
-The output data `samples` is an array of dictionaries where each element of array has the corresponding sample parameter values and the OPF solution. For each input parameter type, the order of array of data is based on the sorted dictionary of the original data in the `base_model`. For examples, `samples[i]["rate_a"]` is a vector containing the thermal rating branch values in sample i and the order is the same order of the keys in `sort(base_model["branch"])`. Or `samples[i]["qd"]` is a vector containing the load reactive power values in sample i and the order is the same order of the keys in `sort(base_model["load"])`.   
+The output data `samples` is an array of dictionaries where each element of array has the corresponding sample parameter values and the OPF solution. For each input parameter type, the order of array of data is based on the sorted dictionary of the original data in the `base_model`. For examples, `samples[i]["rate_a"]` is a vector containing the thermal rating branch values in sample i and the order is the same order of the keys in `sort(base_model["branch"])`. Or `samples[i]["qd"]` is a vector containing the load reactive power values in sample i and the order is the same order of the keys in `sort(base_model["load"])`.
```

Comment:
> sample parameter values and the OPF solution

I thought the solution was not being included because it is (relatively) cheap to calculate? Also, the files are called `input_*`.

Reply:
Or is this referring to the output of `OPFSampler.jl` directly, and not the data we are providing.

Reply:
In which case, this needs a clarifying comment, as there will be a discrepancy there.

Reply:
oh thanks, good catch. I meant to remove mentioning of OPF_ouput but this one slipped under my radar. Will fix

Reply:
Please check if the changes in the new commit clarify things.

Reply:
@AlexRobson

Reply:
@AlexRobson I reorganized things a bit and added more description on the original PowerModels data and the mapping between our samples and that data. Can you please take a look to see if it is more clear?